src/sysext/sysext.c

```diff
@@ -1779,19 +1809,29 @@ static int merge_subprocess(
                 ImageClass image_class,
                 char **hierarchies,
                 bool force,
+                bool always_refresh,
                 int noexec,
                 Hashmap *images,
                 const char *workspace) {
 
         _cleanup_free_ char *host_os_release_id = NULL, *host_os_release_id_like = NULL,
                         *host_os_release_version_id = NULL, *host_os_release_api_level = NULL,
-                        *filename = NULL;
+                        *filename = NULL, *old_origin_content = NULL,
+                        *extensions_origin_content = NULL, *arg_root_resolved = NULL;
```

Comment:
Please don't name this `arg_root_resolved`, we reserve that for the global arguments. `root_resolved` should be sufficient